src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Figure figure;
+
+    public Figure getRandomFigure() {
+        int numOfFigure = random.nextInt(Figure.amountOfFigures);
+        switch (numOfFigure) {
+            case 0:
+                figure = new Circle(colorSupplier.getRandomColor(), random.nextInt(1,10));
```

Comment:
```suggestion
                return new Circle(color, random.nextInt(1,10));
```

create separate method for `random.nextInt(1,10)` and use it for each type of figure